.github/workflows/test-jruby.yml

```
@@ -0,0 +1,22 @@
+name: test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: build (${{ matrix.ruby }} / ${{ matrix.os }})
+    strategy:
+      matrix:
+        ruby: [ 2.7, 2.6, head, jruby-9.3 ]
+        os: [ ubuntu-latest, macos-latest ]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: ${{ matrix.ruby }}
+    - name: Install dependencies
+      run: bundle install
+    - name: Run test
+      run: rake test
```

To use the fancy caching, we can:

```suggestion
        ruby-version: ${{ matrix.ruby }}
        bundler-cache: true # 'bundle install' and cache
    - name: Run test
      run: bundle exec rake test
```

---

I have largely been copying these configs from the existing ones. Perhaps a round of PRs to update the other gems, so this doesn't keep getting propagated without the caching?